### ISSUE TITLE
Address munging

### ISSUE THREAD

**Issue by gpasqualetti** [ai-sf/ai-sf.github.io#11]
Per evitare di essere ulteriormente soggetti a spamming per le mail sarebbe opportuno adottare qualche pratica di address munging o alternative come suggerito su [wikipedia](https://en.wikipedia.org/wiki/Address_munging#Alternatives). Necessaria investigazione sul metodo più efficace e meno invasivo.

**Comment by Foelering** [comment]
http://perso.crans.org/~raffo/email-munging.php
La soluzione 5 in questo link. Implemento tra 2 ore via Liquid.

**Comment by Foelering** [comment]
No, scherzo. Faccio un primo fix dove sostituisco "a mano" ogni @ con l'ASCII. Dopodichè migrerei tutte le info sulle persone linkate su un file di configurazione e farei generare a Jekyll le cose, mungandosi da solo le mail. così si dovrebbe riuscire anche a cambiare più agilmente la composizione del CE in futuro.
